src/crypto/store/memory-crypto-store.js

```diff
@@ -243,10 +243,22 @@ export class MemoryCryptoStore {
         func(this._crossSigningKeys);
     }
 
+    // XXX cache not implemented: I think this is only used in tests
```

Comment:
Can we implement it anyways please? It's going to be awkward when someone goes to use a memory store (which we do export and have docs on, so we can't assume that it's unused) and this bit doesn't work for them.

Reply:
Implemented & tested